core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -301,7 +302,7 @@ class CivilizationInfo {
 
     fun getEquivalentUnit(baseUnitName: String): BaseUnit {
         val baseUnit = gameInfo.ruleSet.units[baseUnitName]
-        if (baseUnit == null) throw UncivShowableException("Unit $baseUnitName doesn't seem to exist!")
+            ?: throw UncivShowableException("Unit $baseUnitName doesn't seem to exist!")
```

Comment:
Same here. I feel the existing was more immediately obvious

Reply:
Doesn't your Android Studio complain? Alternatives would be a `@Suppress("FoldInitializerAndIfToElvis")` or disabling the inspection project-wide - but that would be local I think and not part of what we're syncing as source, so every dev would need their own and we should have documentation?

I'm not overly fond of every inspection myself, but they're there and express opinions of the Kotlin team, right? Ignoring them is like swimming against some social trend - often goes well for a while then you're suddenly an outsider. `?: return` is nice enough, plus the val is actually the simpler type right away - so `?: throw` is no different really... Like the `minByOrNull{}` - ugly as hell but hey we're being forced?

All this of course subject to - _do_ we want Android Studio's 'Problems' window `Alt-6` to be as useful as possible? If it's empty when we start a change then any stuff cropping up here will be immediately obvious....

Reply:
A. There's definitely a bug difference between 'deprecated, do not use' to 'we think this is better but the other is ok too'
B. I think I'll need to rethink my approach to elvis operators. I still don't think it's add transmit, but I'll meditate on this.

Reply:
Meditating is good. Sorry I discovered the 'Problems' window as new toy - but perhaps you agree to start making it useful...